src/lib/components/toast/ToastPositionHelpers.ts

```diff
@@ -0,0 +1,36 @@
+export type ToastPosition =
+  | 'top-left'
+  | 'top-right'
+  | 'top-center'
+  | 'bottom-left'
+  | 'bottom-right'
+  | 'bottom-center';
+
+export const positionOutput: Record<ToastPosition, {}> = {
+  'top-left': {
+    top: '3rem',
+    left: '1rem',
+  },
+  'top-right': {
+    top: '3rem',
+    right: '1rem',
+  },
+  'top-center': {
+    top: '1rem',
+    left: '50%',
+    transform: 'translateX(-50%)',
+  },
```

Comment:
Does those default 'top' position already take the navbar in account ?

Reply:
yes